packages/flutter/lib/src/rendering/table.dart

```diff
@@ -793,8 +793,7 @@ class RenderTable extends RenderBox {
 
   /// Returns the list of [RenderBox] objects that are in the given
   /// column, in row order, starting from the first row.
-  ///
-  /// This is a lazily-evaluated iterable.
```

Comment:
why the removed docs here?

Reply:
Mistake- I was going to refactor this and then realized it was public API. I meant to add the comment back.